docs/build/guides/conventions/workspace.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: Organize contracts with workspace
+hide_table_of_contents: true
+description: Organize your contracts using Cargo workspaces
+---
+
+### Initializing workspace project
+
+Using Cargo's workspace [feature](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html) makes it very convenient to organize your smart contracts in subdirectories of your project's root.
+
+It's very simple to get started using the cli:
+
+```bash
+mkdir soroban-project
+cd soroban-project
+PROJECT_ROOT=.
+stellar contract init . --name add_contract
+```
+
+Running this command will create a root project directory (`soroban-project`) and then initialize a project workspace with a single contract named `add_contract`.
+
+Adding one more contract template to the project can be done using the same command:
+
+```bash
+stellar contract init . --name main_contract
+```
+
+The project tree will look like this:
+
+```
+$ tree
+.
+├── Cargo.toml
+├── contracts
+│   ├── add-contract
+│   │   ├── Cargo.toml
+│   │   ├── Makefile
+│   │   └── src
+│   │       ├── lib.rs
+│   │       └── test.rs
+│   └── main-contract
+│       ├── Cargo.toml
+│       ├── Makefile
+│       └── src
+│           ├── lib.rs
+│           └── test.rs
+└── README.md
+```
+
+Running `contract init` command created a sample contracts subdirectories (located in `./contracts), each contains:
+
+1. Cargo.toml file with all (minimal) necessary dependencies
+2. `Makefile` to easily build contract using `make` command
+3. `src` directory with an example hello-world contract and a simple test.
+
+We can now build our project's contracts with
```

Comment:
Use the word workspace when referring to the workspace, because that's the term the Rust community uses, and the term the tooling describes it as.
```suggestion
We can now build our workspace's contracts with
```